graphics/freetype_font.cpp

```diff
@@ -130,13 +130,13 @@ RawFrame *FreetypeFont::render_string(const char *string) {
 
         //int yd = -(slot->bitmap_top);
         int yd = _baseline - slot->bitmap_top;
-        for (int y = 0; y < slot->bitmap.rows && yd < _h; y++, yd++) {
+        for (int y = 0; (unsigned) y < slot->bitmap.rows && yd < _h; y++, yd++) {
```

Comment:
unsigned int y = 0 and eliminate the cast?

Reply:
Makes sense.